UltraStar Play/Assets/Common/Model/Microphone/MicProfile.cs

```diff
@@ -28,4 +28,19 @@ public MicProfile(string name)
     {
         this.Name = name;
     }
+
+    public int AmplificationMultiplier()
+    {
```

Comment:
In general, Amplification is an int so mapping only a few values is not a good definition. I mean, the AmplificationSlider could just as well allow all positive natural numers.

The value in USDX was given in decibel, which is why I expect it to be in dB as well in UltraStar Play. Is this true, is the amplification in dB? If so, is there no general formula to get from amplification in dB to the AmplificationMultiplier?

Reply:
As alternative, I would put the value of the AmplificationMutliplier directly in the Amplification field and remove the dB unit in the UI. Then it is just some factor.

Reply:
replaced the mapping by proper calculation of the "voltage" amplification multiplier from the field amplification value in the UI. The calculated value is now cached locally.